src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -3290,4 +3290,10 @@ You can manage this in Preferences -&gt; Security.</value>
   <data name="InfectedPackageErrorString" xml:space="preserve">
     <value>Upload Error</value>
   </data>
+  <data name="PrePackagePublishMessage" xml:space="preserve">
+    <value>Your package will be scanned for viruses, and will be available for download once it successfully passes the scan.</value>
```

Comment:
Remove ',' before 'and'

Reply:
The test is as per https://jira.autodesk.com/browse/DYN-4691